### ISSUE TITLE
always gets esp32 reboots

### ISSUE THREAD

**Issue by Radeon91** [darthcloud/BlueRetro#355]
hi.
Tried to build this one but i always gets the esp32 rebooting, check voltage, straigt pins, different firmwware version but the results are always the same.

https://github.com/Micha213/BlueRetro-PS1-2-Receiver

![image](https://user-images.githubusercontent.com/45995754/184649549-97cac176-8d89-4133-9e12-a8c97b1947bb.png)

What can i do?

**Comment by darthcloud** [comment]
Look like power issue like https://github.com/darthcloud/BlueRetro/discussions/170

maybe @Micha213 can help?

**Comment by Micha213** [comment]
Yes, that sounds like a power problem. Did you connect 8v and GND (both) correctly? Can you please take a picture of the installation?

**Comment by Radeon91** [comment]
Tried with 8 volts from ps2 and also 8v and 3.3v from a power supply,
always same result.
